### PR TITLE
Adds focus assessments

### PR DESCRIPTION
**Pull request by argyleink**
2 quiz questions about the focus module

### CODE REVIEW

File: src/site/content/en/learn/css/focus/focus.assess.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      - content: "`<input>`"
+        rationale: "🎉"
+      - content: "`<output>`"
+        rationale: "Try again!"
+      - content: "`<select>`"
+        rationale: "🎉"
+  - type: multiple-choice
+    cardinality: "1+"
+    correctAnswers: "0,1,3"
+    stem: Which of the following input devices cause focus?
```
Comment (argyleink) [inline src/site/content/en/learn/css/focus/focus.assess.yml:25]:
```suggestion
    stem: Which of the following input devices can set focus?
```

File: src/site/content/en/learn/css/focus/focus.assess.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      - content: Gamepad
+        rationale: "Gamepad's often send keyboard events when their buttons are
+        pressed"
+      - content: Keyboard
+        rationale: "Definitely causes focus when used to navigate the web"
+      - content: Mouse
+        rationale: "A mouse requires vision and no longer puts focus on elements
+        when used. All browsers used to put focus on things like buttons when
+        clicked, but that has changed."
+      - content: Screen reader assistive technology
```
Comment (argyleink) [inline src/site/content/en/learn/css/focus/focus.assess.yml:36]:
```suggestion
      - content: Assistive technology (screen reader, switch, etc)
```

**Review by argyleink — COMMENTED** [review]
notes from Rob

**Review by rachelandrew — APPROVED** [review]
Have tested and confirmed that a potato does indeed work on a touchscreen. It's best to find one with a pointy end though.

**Pull request merged**